client/src/modules/notification/components/Notification.jsx

```diff
@@ -7,15 +7,22 @@ import { MessageType as NotificationType } from 'common/constants/enums';
 
 const Notification = ({ data, id, redirect, type }) => (
   <MessageBox type={type}>
-    <FormattedMessage id={id} values={{ data: <em>{data}</em> }} />
+    <FormattedMessage
+      id={id}
+      values={typeof data === 'string' ? { data } : { ...data }}
```

Comment:
1) please extract this above the return
2) Check if we can always return object to avoid such things (if this is a lot of work, create separated task and add this comment into description)

Reply:
1. Done.
2. Task created